packages/emitter-framework/src/python/components/atom/atom.tsx

```diff
@@ -0,0 +1,78 @@
+import { type Children } from "@alloy-js/core";
+import * as py from "@alloy-js/python";
+import { compilerAssert, type Value } from "@typespec/compiler";
+
+/**
+ * Properties for the {@link Atom} component.
+ */
+interface AtomProps {
+  /**
+   * The TypeSpec value to be converted to a Python expression.
+   */
+  value: Value;
+  /**
+   * Hint that this numeric value should be emitted as a float (e.g., 42 -> 42.0).
```

Comment:
would it be possible to pull this out in a `NumericValue` specific interface? I generally don't love properties in the base class that only apply to certain children, but this is more of a nit